src/cdrom/dc/SDL_syscdrom.c

```diff
@@ -28,8 +35,9 @@
 #include <dc/cdrom.h>
 #include <dc/spu.h>
 
+#include "SDL_error.h"
 #include "SDL_cdrom.h"
-#include "../SDL_syscdrom.h"
+#include "SDL_syscdrom.h"
```

Comment:
Is there a point in changing it this way? I see that Makefile.dc is added include paths to specifically support this, but why? <Same for other similar changes in dc-specific sources.)